### PR TITLE
Add link to source mapping video to doc

### PR DESCRIPTION
**Pull request by joelverhagen**
Address https://github.com/NuGet/docs.microsoft.com-nuget/issues/2587

### CODE REVIEW

File: docs/consume-packages/Package-Source-Mapping.md [hunk 1]
```diff
@@ -24,6 +24,10 @@ Older tooling will ignore the Package Source Mapping configuration. To use this
 
 Package Source Mappings will apply to all project types – including .NET Framework – as long as compatible tooling is used.
 
+## Related video
```
Comment (nkolev92) [inline docs/consume-packages/Package-Source-Mapping.md:27]: I love that we're adding this. 

I'm wondering if we can tweak the title a little bit. Should it maybe be a video walkthrough or something like that?

**Review by nkolev92 — COMMENTED** [review]
Wondering if we need a better title.

File: docs/consume-packages/Package-Source-Mapping.md (hunk 1 above)
Comment (joelverhagen) [reply docs/consume-packages/Package-Source-Mapping.md:27]: Sounds good! I didn't know what to put so I stole the heading on https://docs.microsoft.com/en-us/nuget/what-is-nuget. I'll do Video walkthrough.

Comment (joelverhagen) [reply docs/consume-packages/Package-Source-Mapping.md:27]:
```suggestion
## Video walkthough
```

File: docs/consume-packages/Package-Source-Mapping.md [hunk 2]
```diff
@@ -24,6 +24,10 @@ Older tooling will ignore the Package Source Mapping configuration. To use this
 
 Package Source Mappings will apply to all project types – including .NET Framework – as long as compatible tooling is used.
 
+## Video walkthough
```
Comment (joelverhagen) [inline docs/consume-packages/Package-Source-Mapping.md:27]:
```suggestion
## Video walkthrough
```

File: docs/consume-packages/Package-Source-Mapping.md (hunk 1 above)
Comment (joelverhagen) [reply docs/consume-packages/Package-Source-Mapping.md]: image
☠️

**Review by nkolev92 — APPROVED** [review]

**Pull request merged**